.github/workflows/docker.yml

```diff
@@ -37,7 +42,8 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
           tags: |
-            type=ref,event=branch
+            type=sha
+            type=ref, event=branch
```

Comment:
why is there an additional space between `type=ref` and `event=branch`?

Reply:
Fixed, this was introduced by mistake.